autometa/common/external/samtools.py

```diff
@@ -61,8 +89,8 @@ def main(args):
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p')
     parser = argparse.ArgumentParser()
-    parser.add_argument('--bam')
```

Comment:
I think `--out` instead of `--bam` is probably more appropriate. However, there should still be some documentation describing the output as a _bam_ file.

Reply:
You could similarly call the output `file.bam` an `alignment.bam` file. Although this is pretty minor and inconsequential. Up to you